languages/python/concepts/list-methods/about.md

```diff
@@ -0,0 +1,286 @@
+Python allows you to manipulate a [`list`][list] in a lot of ways. A `list` is simple a collection of objects of [type `List`][list std type]. They are mutable, ordered and indexed. Let's look at the methods that are available to manipulate a `list` object.
+
+When you manipulate a list with a list-method, you are changing the properties of the list you pass. That is, **you will alter the list** object that is being used with the list-method. If you do not want to change the original list, you need to copy the list and then work on the copied list.
+
+To begin, you first need a `list` object to use the functions.
+
+```python
+>>> empty_list = list() # (or) empty_list = []
+>>> empty_list
+[]
+```
+
+## Add an Item to the List
+
+Topics to discuss:
+
+1. `append`
+2. `insert`
+3. `extend`
+
+If you want to add an item to an existing list, you use the list-method `append()` for it. As the name represent, it appends the item to the **end** of the list.
+
+```python
+>>> numbers = [1, 2, 3]
+>>> numbers.append(9)
+>>> numbers
+[1, 2, 3, 9]
+```
+
+There is another way to add an item to the list. `insert()` method also gives you the ability to add the item to a particular index in the list.
+
+`insert` takes 2 parameters.
+
+1. index at which you want to add the item
+2. item
```

Comment:
`.insert()` takes 2 parameters:

1. index of the item _before which_ you want the new item to appear
2. the item to be inserted